cardano-cli/src/Cardano/CLI/Run/Legacy/Governance.hs

```diff
@@ -127,79 +126,6 @@ runGovernanceCmd = \case
   GovernanceVerifyPoll poll metadata mOutFile ->
     runGovernanceVerifyPoll poll metadata mOutFile
 
-runGovernanceMIRCertificatePayStakeAddrs
-  :: AnyShelleyBasedEra
-  -> Shelley.MIRPot
-  -> [StakeAddress] -- ^ Stake addresses
-  -> [Lovelace]     -- ^ Corresponding reward amounts (same length)
-  -> File () Out
-  -> ExceptT GovernanceCmdError IO ()
-runGovernanceMIRCertificatePayStakeAddrs (AnyShelleyBasedEra sbe) mirPot sAddrs rwdAmts oFp = do
-
-    unless (length sAddrs == length rwdAmts) $
-      left $ GovernanceCmdMIRCertificateKeyRewardMistmach
-               (unFile oFp) (length sAddrs) (length rwdAmts)
-
-    let sCreds  = map stakeAddressCredential sAddrs
-        mirTarget = Ledger.StakeAddressesMIR
-                      $ Map.fromList [ (toShelleyStakeCredential scred, Ledger.toDeltaCoin (toShelleyLovelace rwdAmt))
-                                     | (scred, rwdAmt) <- zip sCreds rwdAmts
-                                     ]
-    mirReq <- hoistEither $ createMirCertificateRequirements sbe mirPot $ obtainEraCryptoConstraints sbe  mirTarget
-    let mirCert = makeMIRCertificate mirReq
-
-    firstExceptT GovernanceCmdTextEnvWriteError
-      . newExceptT
-      $ writeLazyByteStringFile oFp
-      $ textEnvelopeToJSON (Just mirCertDesc) mirCert
-  where
-    mirCertDesc :: TextEnvelopeDescr
-    mirCertDesc = "Move Instantaneous Rewards Certificate"
-
-createMirCertificateRequirements
-  :: ShelleyBasedEra era
-  -> Ledger.MIRPot
-  -> Ledger.MIRTarget (EraCrypto (ShelleyLedgerEra era))
-  -> Either GovernanceCmdError (MirCertificateRequirements era)
-createMirCertificateRequirements sbe mirPot mirTarget =
-  case sbe of
-    ShelleyBasedEraShelley -> do
-      return $ MirCertificateRequirements ShelleyToBabbageEraShelley mirPot mirTarget
-    ShelleyBasedEraAllegra -> do
-      return $ MirCertificateRequirements ShelleyToBabbageEraAllegra mirPot mirTarget
-    ShelleyBasedEraMary -> do
-      return $ MirCertificateRequirements ShelleyToBabbageEraMary mirPot mirTarget
-    ShelleyBasedEraAlonzo -> do
-      return $ MirCertificateRequirements ShelleyToBabbageEraAlonzo mirPot mirTarget
-    ShelleyBasedEraBabbage -> do
-      return $ MirCertificateRequirements ShelleyToBabbageEraBabbage mirPot mirTarget
-    ShelleyBasedEraConway ->
-      Left ShelleyGovernanceCmdMIRCertNotSupportedInConway
```

Comment:
This function is gone because there is not longer a runtime error due to `--conway-era` flag being used.  This flag does not exist for commands that don't work for Conway era.